GitCommands/Git/GitGpgController.cs

```diff
@@ -232,13 +232,14 @@ public string GetCommitVerificationMessage(GitRevision revision)
                 return null;
             }
 
-            var module = GetModule();
             GitArgumentBuilder args = new("verify-tag")
             {
                 { raw, "--raw" },
                 tagName
             };
-            return module.GitExecutable.GetOutput(args);
+
+            // '--raw' returns info in stderr
```

Comment:
```suggestion
            // 'verify-tag' returns info in stderr
```
https://github.com/gitextensions/gitextensions/pull/10262#discussion_r996409350